content/pages/roadmap.html

```diff
@@ -16,25 +16,146 @@ <h4><i class="fa fa-map"></i> <b>Roadmap</b></h2>
 
                 <p>If you have an opinion about where iRODS should go next, I encourage you to participate in this open software development process. Join the <a href="https://groups.google.com/forum/#!forum/irod-chat">iRODS chat Google Group</a>. Raise issues on <a href="https://github.com/irods/irods">GitHub</a>. And if you are a <a href="{filename}/pages/consortium-membership.html">Consortium Member</a>, get involved in the Technology Working Group.</p>
 
-                <p>Terrell Russell<br/>
+                <p>Kory Draughn<br/>
```

Comment:
Perhaps I should restore this?

Seems weird to have my name attached to this section.

Reply:
no?   you're the chief technologist, and it says 'I' in there... so someone is saying it :)

Reply:
Ok